heartbeat/iSCSITarget.in

```diff
@@ -391,6 +391,7 @@ iSCSITarget_start() {
 			done
 		else
 			ocf_run targetcli /iscsi/${OCF_RESKEY_iqn}/tpg1/ set attribute authentication=0 demo_mode_write_protect=0 generate_node_acls=1 cache_dynamic_acls=1 || exit $OCF_ERR_GENERIC
+			ocf_run targetcli /iscsi/${OCF_RESKEY_iqn}/tpg1/ set auth userid=${OCF_RESKEY_incoming_username} password=${OCF_RESKEY_incoming_password} || exit $OCF_ERR_GENERIC
```

Comment:
You should add `if [ -n "${OCF_RESKEY_incoming_username}" ]; then` around it to ensure that it's also working when used without user/pass.